core/lib/spree/core/engine.rb

```diff
@@ -48,15 +48,11 @@ class Engine < ::Rails::Engine
       initializer 'spree.core.initialize_subscribers' do |app|
         app.reloader.to_prepare do
           Spree::Event.require_subscriber_files
-          Spree::Event.subscribers.each_key do |klass|
-            klass.constantize.subscribe!
-          end
+          Spree::Event.subscribe_subscribers!
```

Comment:
The more I look into this and the more I think we need another class called `Spree::Event::Subscribers` that acts as a collection of subscribers. No matter if it's hash inside but the interface will probably look better, having something like:

`Spree::Event.subscribers.subscribe_all`. 

WDYT?

Reply:
sure, makes sense! I'll work on your suggestion 👍